### PR TITLE
Added logic to separate past events

### PR DESCRIPTION
**Pull request by x3N1GM4x**
Changed the "Available Kits" page so that any kits with an "Expires On" date in the past are shifted down into a "Past Kits" section and when opened do not show the purchasing section at the bottom, so people cannot attempt to purchase previous kits.

Would appreciate someone giving this a thorough once over to make sure I've not done anything dumb or missed anything - look forward to seeing all my bugs found! ;)

### CODE REVIEW

**Comment by x3N1GM4x** [comment]
[P1: Enforce expiry date for purchasing event kits](https://trello.com/c/vfIBchb2/60-p1-enforce-expiry-date-for-purchasing-event-kits)

**Comment by binarymelon** [comment]
It's not really a result of these changes, but it might also be a good idea to add some validation in [PaypalController.php](https://github.com/x3N1GM4x/project-nisei/blob/event-expiry/app/Http/Controllers/Web/PaypalController.php) to prevent purchasing expired kits.

**Comment by binarymelon** [comment]
I'll try to give it a test tomorrow.  Just giving the code a scan, it looks good to me now.

File: app/Http/Controllers/Web/PaypalController.php [hunk 1]
```diff
@@ -69,6 +69,10 @@ public function postPaymentWithPaypal(Request $request)
             Session::put('error','Unable to order more than 1 kit');
             return redirect()->back();
         }
+        if ($event->expires_on < time()) {
+            Session::put('error','Kit no longer on sale');
```
Comment (x3N1GM4x) [inline app/Http/Controllers/Web/PaypalController.php]: Having now tested this on my local environment, I get the "Kit no longer on sale" response even when I attempt to purchase a kit with an Expires On date in the future. Is anyone else able to take a look and figure out what's wrong?

File: app/Http/Controllers/Web/PaypalController.php [hunk 2]
```diff
@@ -69,6 +69,10 @@ public function postPaymentWithPaypal(Request $request)
...
+            return redirect()->back();
+        }
```
Comment (x3N1GM4x) [inline app/Http/Controllers/Web/PaypalController.php]: Having now tested this on my local environment, I get the "Kit no longer on sale" response even when I attempt to purchase a kit with an Expires On date in the future. Is anyone else able to take a look and figure out what's wrong?

**Pull request merged**